index.html

```diff
@@ -4072,6 +4072,11 @@ <h3>Co-signed</h3>
               <td>Individual</td>
               <td>Development; open-source community efforts</td>
             </tr>
+	    <tr>
+	       <td><a href="https://github.com/AMitBehr>Aidan Behrens</a></td>
```

Comment:
You're missing quotation mark at the end of the link. Please fix that and push another commit.